pkg/storkctl/clusterpair.go

```diff
@@ -623,7 +623,7 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 	createClusterPairCommand.Flags().StringVar(&azureAccountKey, "azure-account-key", "", "Account key for Azure")
 	// Google
 	createClusterPairCommand.Flags().StringVar(&googleProjectID, "google-project-id", "", "Project ID for Google")
-	createClusterPairCommand.Flags().StringVar(&googleJSONKey, "google-json-key", "", "Json key for Google")
+	createClusterPairCommand.Flags().StringVar(&googleJSONKey, "google-json-key-file", "", "Json key for Google")
```

Comment:
We can be more explicit and say 
--google-key-file-path

Reply:
Done